release-notes/7.0/supported-os.md

```diff
@@ -28,13 +28,13 @@ OS                                    | Version                 | Architectures
 OS                                    | Version               | Architectures     | Lifecycle
 --------------------------------------|-----------------------|-------------------|----------
 [Alpine Linux][Alpine]                | 3.15+                 | x64, Arm64, Arm32 | [Alpine][Alpine-lifecycle]
-[CentOS Linux][CentOS]                | 7                     | x64               | [CentOS][CentOS-lifecycle]
-[CentOS Stream Linux][CentOS]         | 8                     | x64               | [CentOS][CentOS-lifecycle]
+[CentOS][CentOS]                      | 7                     | x64               | [CentOS][CentOS-lifecycle]
+[CentOS Stream][CentOS]               | 8+                    | x64               | [CentOS][CentOS-lifecycle]
 [Debian][Debian]                      | 10+                   | x64, Arm64, Arm32 | [Debian][Debian-lifecycle]
 [Fedora][Fedora]                      | 36+                   | x64               | [Fedora][Fedora-lifecycle]
 [openSUSE][OpenSUSE]                  | 15+                   | x64               | [OpenSUSE][OpenSUSE-lifecycle]
-[Oracle Linux][Oracle-Linux]          | 7+                    | x64               | [Oracle][Oracle-lifecycle]
-[Red Hat Enterprise Linux][RHEL]      | 7+                    | x64, Arm64        | [Red Hat][RHEL-lifecycle]
+[Oracle Linux][Oracle-Linux]          | 8+                    | x64               | [Oracle][Oracle-lifecycle]
```

Comment:
Is this really 8+ as well? Oracle Linux 7 is only EOL in July 2024. So I would expect it to be supported on .NET 6 and 7.

https://learn.microsoft.com/en-us/dotnet/core/install/linux is sadly missing a page for Oracle Linux, so I can't check that.